test/contracts/sophia_2/crypto.aes

```diff
@@ -15,6 +15,14 @@ contract CryptoTest =
   function sha256  (x : complex) : hash = Crypto.sha256(x)
   function blake2b (x : complex) : hash = Crypto.blake2b(x)
 
+  function sha3_b17    (x : bytes(17)) : hash = Crypto.sha3(x)
```

Comment:
Is this one necessary? Didn't the tests exclude minerva?

Reply:
Huh, how does it compile on minerva in that case..? 🤔

Reply:
No, I just meant the bit with the newly added functions?